docs/contributing/setup.md

```diff
@@ -113,6 +113,16 @@ is as follows:
 * Run `yarn build:dev` to create a development build of the app.
 * Run `yarn start` to launch the application. Changes will be compiled in the
   background. The app can then be reloaded to see the changes (Ctrl/Command+R).
+  
+**Tip**: Use `screen` to avoid filling your terminal with logging output:
+
+```shellsession
+$ screen -S "desktop" "yarn start" # -S sets the name of the session; you can pick anything
```

Comment:
On macOS Sierra in iTerm2 I see the screen detach and then a `Cannot exec 'yarn start': No such file or directory` in the bottom-left, before it gives up:

<img width="395" src="https://user-images.githubusercontent.com/359239/32636545-68f2fbd6-c609-11e7-92a5-3c74a7b825fe.png">

Reply:
> Cannot exec 'yarn start': No such file or directory

Does `screen -S "desktop" yarn start` work? (no quotes around `yarn start`)

Reply:
Yeah, that's the issue.